src/translations/fr.json

```diff
@@ -9,24 +9,43 @@
     "private": "privé",
     "public": "public",
     "emptyDir": "Dossier vide",
-    "accessRights": "Droits d'accès",
 
     "addNewStudy": "Ajouter une nouvelle étude",
     "addNewStudyDescription": "Pour ajouter une étude, vous devez saisir les informations suivantes : ",
+    "renameStudy": "Renommer l'étude",
+    "renameStudyMsg": "Nouveau nom de l'étude : ",
+    "renameStudyNotAllowedError": "Vous ne pouvez pas renommer une étude dont vous n'êtes pas propriétaire",
+    "renameStudyNotFoundError": "L'étude n'a pas été trouvé",
```

Comment:
`renameStudyNotFoundError` and `modifyAccessRightsNotFoundError` have the same string.
Maybe use `StudyNotFoundError` instead